classes/class-uagb-post-assets.php

```diff
@@ -385,11 +385,14 @@ public function allow_assets_generation() {
 	 * @since 1.23.0
 	 */
 	public function enqueue_scripts() {
-		global $_wp_current_template_content;
-		$blocks = parse_blocks( $_wp_current_template_content );
-		// Global Required assets.
-		// If the current template has content and contains blocks, execute this code block.
-		if ( has_blocks( $this->post_id ) || ( $_wp_current_template_content && has_blocks( $blocks ) ) ) {
+			$blocks = array();
+		if ( function_exists( 'wp_is_block_theme' ) && wp_is_block_theme() ) {
```

Comment:
Suggestion - Needs to be tested if implemented.
```suggestion
		if ( UAGB_Admin_Helper::is_block_theme() ) {
```

Reply:
We can mark this as resolved, and implement this later if needed.
CC - @adi3890

Reply:
Done in [this commit](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3509/commits/a1a41df1f7d7565015bfce0d4d2b47b7690eaf96).

Props: @adi3890.

Reply:
Resolved - https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3511